Curve-finance-suite/New-Fee/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "forta-agent-starter",
+  "version": "0.0.1",
+  "description": "Forta Agent Typescript starter project",
```

Comment:
Update `name` and `description`

Reply:
We usually use the `alertId`(in lowercase) used in the agent as `name`.

Reply:
Addressed in latest commit